### PR TITLE
[W5.11][W09-A2]Ow Zhen Wei

### PR DESCRIPTION
**Pull request by whenzei**
Add new sort command to sort entries in the address book by the lexicographical order of name.

### CODE REVIEW

File: doc/UserGuide.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ Examples:
 * `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
 * `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
 
+### Sorting all persons : `sort`
+Sorts the list of all persons in the address book, and shows the sorted list<br>
+Format: `list`
```
Comment (jethrokuan) [inline doc/UserGuide.md]: this looks incorrect, should be `sort`.

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Sorts all persons in the address book
+ */
+public class SortCommand extends Command{
```
Comment (eugenepeh) [inline src/seedu/addressbook/commands/SortCommand.java:12]: missing space before `left curly`

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (eugenepeh) [inline src/seedu/addressbook/parser/Parser.java:14]: remember to disable intellij auto import wildcard
[Why is wild card import bad?](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad)

**Review by eugenepeh — COMMENTED** [review]
Good job on completing the activity.
Some comments added. Please close the PR after reading the comments.

**Pull request closed without merging**